contrib/pa-pass

```diff
@@ -3,31 +3,16 @@
 # pa-pass - a simple tool to migrate passwords from pass to pa
 
 pw_migrate() {
-    find $1 -name '*.gpg' | while read -r passfile; do
-        local agefile=$(echo "$passfile" | awk -F"$1" '{print $2}' | sed 's/gpg/age/')
-	local agefolder=$(dirname $agefile)
-	mkdir -p $PA_DIR$agefolder
-	gpg2 -d $passfile | age --encrypt -R "$recipients_file" -o "$PA_DIR$agefile"
-	printf '%s\n' "Saved '$agefile' to the store."
-    done	
-}
-
-
-glob() {
-    # This is a simple wrapper around a case statement to allow
-    # for simple string comparisons against globs.
-    #
-    # Example: if glob "Hello World" '* World'; then
-    #
-    # Disable this warning as it is the intended behavior.
-    # shellcheck disable=2254
-    case $1 in $2) return 0 ;; esac
-    return 1
-}
-
-die() {
-    printf 'error: %s.\n' "$1" >&2
-    exit 1
+    # Create pa store if it doesn't exist
+    pa list >/dev/null
```

Comment:
This will run the backward compatibility logic like https://github.com/biox/pa/blob/325f63b1d10252bd0f62bf4cfa61b06cf17ea6ac/pa#L200
but... anyway.

Reply:
well if you don't have a pa store this is a no-op, and if you do, this is a good thing